### PR TITLE
Session 2 Exercises

### PR DESCRIPTION
**Pull request by weidnem**
Not fancy. Git 'er done.

### CODE REVIEW

**Comment by codedragon** [comment]
So, grid.py is empty. Was there code that you wanted us to look at?

**Comment by weidnem** [comment]
Updated with actual grid exercise and fib exercise.

File: students/weidnem/session2/grid.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            pos += 1
+    else:
+        print()
+
+
+
+printgrid()
+
+
+
```
Comment (codedragon) [inline students/weidnem/session2/grid.py]: This is a good start. Keep in mind that you can multiply strings. How would you change this to make the grid the size the user requests?

File: students/weidnem/session2/series.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+def get_next(series,n):
+    if len(series) != n:
+        series.append(series[-2]+series[-1])
+        get_next(series,n)
+        
+    else:
+        print(series)
+
+
+fibonacci(9)
```
Comment (codedragon) [inline students/weidnem/session2/series.py]: This is a great. Note that the exercise was to return just the nth number in the series. Do you see how you can adapt this code to do the lucas exercise?

**Pull request merged**